test/Microsoft.SqlTools.ServiceLayer.TestDriver/Utility/TestRunner.cs

```diff
@@ -23,9 +23,8 @@ public class TestRunner
                     bool containsTestName = testName.Contains(".");
                     var className = containsTestName ? testName.Substring(0, testName.LastIndexOf('.')) : testName;
                     var methodName = containsTestName ? testName.Substring(testName.LastIndexOf('.') + 1) : null;
-
-
-                    var type = Type.GetType(testNamespace + className);
+                    Assembly assembly = Assembly.GetEntryAssembly();
```

Comment:
the current type (Testrunner) doesn't have those test methods because the methods are in different assembly (PerfTest)

---
In reply to: [88958836](https://github.com/Microsoft/sqltoolsservice/pull/155#discussion_r88958836) [](ancestors = 88958836)